src/System.Windows.Forms.Primitives/tests/TestUtilities/Metafiles/Validators/RepeatValidator.cs

```
@@ -23,9 +23,11 @@ public RepeatValidator(IEmfValidator validator, int count)
 
         public void Validate(ref EmfRecord record, DeviceContextState state, out bool complete)
         {
+            // TODO: should we throw if we're at a negative count?
```

@JeremyKuhne what do you think?

---

Should Assert that the count > 0. That will report properly.

---

Hmm. Actually, no, we should throw `InvalidOperationException` for -1. We should never be keeping this in scope if we hit zero.